example-app/src/main/cpp/native-lib.cpp

```diff
@@ -1,12 +1,21 @@
 #include <jni.h>
 #include <signal.h>
+#include <unistd.h>
 #include "backtrace-android.h"
 
+void * volatile always_null;
+
+void anotherCrash()
+{
+//    memset(always_null, 0x42, 1 << 20);
```

Comment:
do we need that or can be removed?

Reply:
For native crashes it can be useful to test different ways of crashing. I will add a comment making this more clear, thank you!

Reply:
@rqbacktrace  does it make sense to move this function to different .cpp file? Are you sure this should be a part of the `native-lib` source code? I think it belongs to "helpers" or "utils"?

Reply:
Please keep in mind this code will end up in someone production code. We should be super careful with code that we're exposing/adding to libraries.